packages/mdc-chips/README.md

```diff
@@ -52,13 +52,39 @@ npm install --save @material/chips
 </div>
 ```
 
+#### Leading and Trailing Icons
+
+You can optionally add a leading icon (i.e. thumbnail) and/or a trailing icon to a chip. To add an icon, add the relevant class (either `mdc-chip--with-leading-icon` or `mdc-chip--with-trailing-icon`) to the chip element, add an `i` element with your preferred icon, and give it a class of `mdc-chip__icon`. If you're adding a trailing icon, also give the icon a class of `mdc-chip__icon--trailing`.
+
+##### Leading icon
+
+```html
+<div class="mdc-chip mdc-chip--with-leading-icon">
+  <i class="material-icons mdc-chip__icon" tabindex="0">event</i>
+  <div class="mdc-chip__text">Add to calendar</div>
+</div>
+```
+
+##### Trailing icon
+
+```html
+<div class="mdc-chip mdc-chip--with-trailing-icon">
+  <div class="mdc-chip__text">Jane Smith</div>
+  <i class="material-icons mdc-chip__icon mdc-chip__icon--trailing" tabindex="0">cancel</i>
+</div>
+```
+
 ### CSS Classes
 
 CSS Class | Description
 --- | ---
+`mdc-chip-set` | Mandatory. Indicates the set that the chip belongs to
 `mdc-chip` | Mandatory.
-`mdc-chip__text` | Mandatory. Indicates the text content of the chip.
-`mdc-chip-set` | Mandatory. Indicates the set that the chip belongs to.
+`mdc-chip--with-leading-icon` | Optional. Indicates the chip contains a leading icon
+`mdc-chip--with-trailing-icon` | Optional. Indicates the chip contains a trailing icon
+`mdc-chip__text` | Mandatory. Indicates the text content of the chip
+`mdc-chip__icon` | Optional. Indicates a leading or trailing icon in the chip
```

Comment:
Can we mention that "leading" is the default unless `--trailing` is used?

Reply:
Updated it so either `--leading` or `--trailing` is mandatory, which also removes the need for `mdc-chip--with-leading-icon` and `mdc-chip--with-trailing-icon`.